front-end/src/environments/environment.cloud.gov.dev.ts

```diff
@@ -11,6 +11,11 @@ export const environment = {
   fecApiUrl: 'https://api.open.fec.gov/v1/',
   fecApiKey: 'DVoLzo07NBfrbDZPj0LJs3PS0GIRL3fk4eOp7Zo6',
   userCanSetFilingFrequency: true,
+  loginDotGovAuthUrl: 'https://fecfile-web-api-dev.app.cloud.gov/oidc/authenticate',
+  loginDotGovLogoutUrl: 'https://fecfile-web-api-dev.app.cloud.gov/oidc/logout/',
```

Comment:
Looks like we need the equivalent loginDotGov environment variables set in the stage and prod env files as well.